subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/AbstractTestTask.java

```diff
@@ -120,6 +123,7 @@ public AbstractTestTask() {
         testOutputListenerBroadcaster = listenerManager.createAnonymousBroadcaster(TestOutputListener.class);
         testListenerBroadcaster = listenerManager.createAnonymousBroadcaster(TestListener.class);
         binaryResultsDirectory = getProject().getObjects().directoryProperty();
+        successWithoutTest = getProject().getObjects().property(Boolean.class).convention((Boolean)null);
```

Comment:
:x: No need for a conventional value
```suggestion
        successWithoutTest = getProject().getObjects().property(Boolean.class);
```